bluepyopt/ephys/locations.py

```diff
@@ -521,10 +521,11 @@ def set_sec_index(self, icell=None):
                 for section in getattr(icell, self.seclist_name)
             ]
         )
-        if self.direction == 'radial':
-            self.sec_index = int(np.argmax(np.linalg.norm(points, axis=1)))
-        else:
-            self.sec_index = int(np.argmax(points.dot(self.direction)))
+        if len(points):
```

Comment:
Can you be explicit here, prefer if statements with explicit equations

Reply:
you mean adding >0?